### PR TITLE
chore: added emojis, fixed spelling errors and grammar errors, spaced text out, added bullet points and made some text italic and bold

### PR DESCRIPTION
**Pull request by HareerSakhi**
I have completed folder 9 of the english normal-dungeon. Thanks

### CODE REVIEW

**Comment by ming-tsai** [comment]
Hi,
Could you remove the new space in the middle of the list options?
You have like this
```md
- Option1

- Option2

- OptionN+1
```
but the best way is
```md
- Option1
- Option2
- OptionN+1
```

**Comment by HareerSakhi** [comment]
Thanks for your comment. I have fixed this.

File: english/normal-dungeon/9/3-CA.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-Dust covers the scattered papers that lie upon the desk, all written in a language you cannot read. Nothing of note seems to be on it or in it's drawers.
+Dust covers the scattered papers 📰 that lie upon the desk, all written in a language you cannot read 😐. No notes seems to be on the piece of papers 📰 or in it's drawers.
 
-[📚 Examine the bookshelves?](3-CB.md)
+***What will you do next?***
 
-[🔙 Go back](2.md)
+-[ Examine the bookshelves? 📚](3-CB.md)
```
Comment (ming-tsai) [inline english/normal-dungeon/9/3-CA.md:5]: Missing a space
```suggestion
- [Examine the bookshelves? 📚](3-CB.md)
```

Comment (HareerSakhi) [reply english/normal-dungeon/9/3-CA.md]: Complete, thanks

**Comment by HareerSakhi** [comment]
Hi, fixed the error with the space and bullet point. Thanks

**Review by ming-tsai — APPROVED** [review]

**Comment by ming-tsai** [comment]
Nice work @HareerSakhi

**Pull request merged**